interfaces/uss_qualifier/uss_qualifier.yaml

```diff
@@ -1,93 +1,80 @@
                     type: string
                 user_config:
                     description: >-
-                        User authentication configuration.
+                        JSON string of configuration specifying which tests to run and how to run them.
                     type: string
         CreateTestRunRequest:
             type: object
             required:
-                - flight_records
-                - auth_spec
-                - user_config
+                - specification
             properties:
-                flight_records:
-                    description: >-
-                        Comma separated list of flight record file names. Files should be uploaded by upload method before running the tests.
-                    type: string
-                auth_spec:
-                    description: >-
-                        User authentication specification.
-                    type: string
-                user_config:
-                    description: >-
-                        User authentication configuration.
-                    type: string
-        FlightRecordsUploadKmlResponse:
+                specification:
+                    $ref: "#/components/schemas/TestRunSpecification"
+        CreateKmlImportJobResponse:
             type: object
             properties:
-                id:
+                kml_import_job_id:
                     type: string
                 status_message:
                     type: string
                     example: "A task has been started in the background."
                     description: >-
                         Status of currently running task.
-                files:
-                    type: array
-                    items:
-                        type: string
-                    description: >-
-                        KML files uploaded to generate flight records.
                 background_tasks:
                     type: array
                     items:
-                        type: string
+                        type: TaskID
                     description: >-
                         A list of task ids running in the background. The status of each such task can be checked through /api/tasks/<task_id>.
         FlightRecordsUploadJsonResponse:
             type: object
             properties:
-                id:
-                    type: string
-                status_message:
-                    type: string
-                    example: "A task has been started in the background."
-                    description: >-
-                        Status of currently running task.
-                files:
+                flight_record_ids:
                     type: array
                     items:
-                        type: string
-        TestRunResponse:
+                        type: FlightRecordID
+        GetTestRunResponse:
+            type: object
+            properties:
+                test_run:
```

Comment:
The original idea was that all responses should be their own objects so that metadata could be easily attached in the future, but [Google AIP](https://google.aip.dev/131) recommends returning the object directly so I'll make that change.